### ISSUE TITLE
Docker installation not working

### ISSUE THREAD

**Issue by WitteStier** [directus/directus#3282]
Hi,

I tried to install Directus following the [docker installation guide](https://docs.directus.io/guides/installation/docker.html#docker-compose).

When executing `docker-compose up` I get the following output

```
Creating directus_directus_1 ... done
Creating directus_cache_1    ... done
Creating directus_database_1 ... done
Attaching to directus_directus_1, directus_database_1, directus_cache_1
cache_1     | 1:C 03 Dec 2020 10:15:14.576 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
cache_1     | 1:C 03 Dec 2020 10:15:14.576 # Redis version=6.0.9, bits=64, commit=00000000, modified=0, pid=1, just started
cache_1     | 1:C 03 Dec 2020 10:15:14.576 # Warning: no config file specified, using the default config. In order to specify a config file use redis-server /path/to/redis.conf
cache_1     | 1:M 03 Dec 2020 10:15:14.578 * Running mode=standalone, port=6379.
cache_1     | 1:M 03 Dec 2020 10:15:14.578 # Server initialized
cache_1     | 1:M 03 Dec 2020 10:15:14.578 # WARNING overcommit_memory is set to 0! Background save may fail under low memory condition. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.
cache_1     | 1:M 03 Dec 2020 10:15:14.578 * Ready to accept connections
database_1  | The files belonging to this database system will be owned by user "postgres".
database_1  | This user must also own the server process.
database_1  | 
database_1  | The database cluster will be initialized with locale "en_US.utf8".
database_1  | The default database encoding has accordingly been set to "UTF8".
database_1  | The default text search configuration will be set to "english".
database_1  | 
database_1  | Data page checksums are disabled.
database_1  | 
database_1  | fixing permissions on existing directory /var/lib/postgresql/data ... ok
database_1  | creating subdirectories ... ok
database_1  | selecting dynamic shared memory implementation ... posix
database_1  | selecting default max_connections ... 100
database_1  | selecting default shared_buffers ... 128MB
database_1  | selecting default time zone ... Etc/UTC
database_1  | creating configuration files ... ok
directus_1  | 10:15:14 ✨ Initializing...
database_1  | running bootstrap script ... ok
directus_1  | 10:15:14 ✨ Checking database connection
database_1  | performing post-bootstrap initialization ... ok
database_1  | syncing data to disk ... ok
database_1  | 
database_1  | 
database_1  | Success. You can now start the database server using:
database_1  | 
database_1  |     pg_ctl -D /var/lib/postgresql/data -l logfile start
database_1  | 
database_1  | initdb: warning: enabling "trust" authentication for local connections
database_1  | You can change this by editing pg_hba.conf or using the option -A, or
database_1  | --auth-local and --auth-host, the next time you run initdb.
database_1  | waiting for server to start....2020-12-03 10:15:15.195 UTC [47] LOG:  starting PostgreSQL 12.5 (Debian 12.5-1.pgdg100+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 8.3.0-6) 8.3.0, 64-bit
database_1  | 2020-12-03 10:15:15.195 UTC [47] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
database_1  | 2020-12-03 10:15:15.204 UTC [48] LOG:  database system was shut down at 2020-12-03 10:15:15 UTC
database_1  | 2020-12-03 10:15:15.206 UTC [47] LOG:  database system is ready to accept connections
database_1  |  done
database_1  | server started
database_1  | CREATE DATABASE
database_1  | 
database_1  | 
database_1  | /usr/local/bin/docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/*
database_1  | 
database_1  | 2020-12-03 10:15:15.369 UTC [47] LOG:  received fast shutdown request
database_1  | waiting for server to shut down....2020-12-03 10:15:15.369 UTC [47] LOG:  aborting any active transactions
database_1  | 2020-12-03 10:15:15.370 UTC [47] LOG:  background worker "logical replication launcher" (PID 54) exited with exit code 1
database_1  | 2020-12-03 10:15:15.370 UTC [49] LOG:  shutting down
database_1  | 2020-12-03 10:15:15.374 UTC [47] LOG:  database system is shut down
database_1  |  done
database_1  | server stopped
database_1  | 
database_1  | PostgreSQL init process complete; ready for start up.
database_1  | 
database_1  | 2020-12-03 10:15:15.481 UTC [1] LOG:  starting PostgreSQL 12.5 (Debian 12.5-1.pgdg100+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 8.3.0-6) 8.3.0, 64-bit
database_1  | 2020-12-03 10:15:15.481 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
database_1  | 2020-12-03 10:15:15.481 UTC [1] LOG:  listening on IPv6 address "::", port 5432
database_1  | 2020-12-03 10:15:15.482 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
database_1  | 2020-12-03 10:15:15.492 UTC [65] LOG:  database system was shut down at 2020-12-03 10:15:15 UTC
database_1  | 2020-12-03 10:15:15.494 UTC [1] LOG:  database system is ready to accept connections
directus_1  | 10:15:16 ✨ Database installed
directus_1  | 10:15:16 ✨ Creating administrator role
```
Also the same result when passing the admin vars:
`env DIRECTUS_ADMIN_EMAIL=admin@example.com DIRECTUS_ADMIN_PASSWORD=d1r3ctu5 docker-compose up`

When I stop the containers ( Ctrl+c) and start it again, Directus seems to start, however, when I enter the configured email and password I get a `Wrong username or password` error.
```
Attaching to directus_database_1, directus_cache_1, directus_directus_1
database_1  | 
database_1  | PostgreSQL Database directory appears to contain a database; Skipping initialization
database_1  | 
database_1  | 2020-12-03 10:36:29.141 UTC [1] LOG:  starting PostgreSQL 12.5 (Debian 12.5-1.pgdg100+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 8.3.0-6) 8.3.0, 64-bit
database_1  | 2020-12-03 10:36:29.202 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
database_1  | 2020-12-03 10:36:29.202 UTC [1] LOG:  listening on IPv6 address "::", port 5432
database_1  | 2020-12-03 10:36:29.206 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
database_1  | 2020-12-03 10:36:29.227 UTC [24] LOG:  database system was shut down at 2020-12-03 10:35:22 UTC
database_1  | 2020-12-03 10:36:29.232 UTC [1] LOG:  database system is ready to accept connections
cache_1     | 1:C 03 Dec 2020 10:36:29.264 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
cache_1     | 1:C 03 Dec 2020 10:36:29.264 # Redis version=6.0.9, bits=64, commit=00000000, modified=0, pid=1, just started
cache_1     | 1:C 03 Dec 2020 10:36:29.264 # Warning: no config file specified, using the default config. In order to specify a config file use redis-server /path/to/redis.conf
cache_1     | 1:M 03 Dec 2020 10:36:29.266 * Running mode=standalone, port=6379.
cache_1     | 1:M 03 Dec 2020 10:36:29.266 # Server initialized
cache_1     | 1:M 03 Dec 2020 10:36:29.266 # WARNING overcommit_memory is set to 0! Background save may fail under low memory condition. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.
cache_1     | 1:M 03 Dec 2020 10:36:29.266 * Loading RDB produced by version 6.0.9
cache_1     | 1:M 03 Dec 2020 10:36:29.266 * RDB age 67 seconds
cache_1     | 1:M 03 Dec 2020 10:36:29.266 * RDB memory usage when created 0.78 Mb
cache_1     | 1:M 03 Dec 2020 10:36:29.266 * DB loaded from disk: 0.000 seconds
cache_1     | 1:M 03 Dec 2020 10:36:29.266 * Ready to accept connections
directus_1  | 10:36:29 ✨ Initializing...
directus_1  | 10:36:29 ✨ Checking database connection
directus_1  | 10:36:30 ✨ Server started at port 8055
```

## Version Info
__Directus__: `v9.0.0-rc.19`
__Docker__: `Docker version 19.03.8`
__Compose__: `docker-compose version 1.25.0`

**Comment by Michoern** [comment]
Hey @WitteStier,
I have the same issue. For me I fixed it with the following workaround: 

Get to your shell and use `docker exec -t -i CONTAINER_ID /bin/sh`.
Then use the function from https://docs.directus.io/reference/command-line-interface.html
For me npx got caught to a certain case so I had to forced it to exit.
If you have the same issue break it into two parts:

First use `npx directus roles create --name Administrator --admin`.
It will print something like this `9bc8c35d-6145-4595-8287-1895c747cb28` to your console, this is your ROLE_ID. 
After it is printed the process stuck by me and I had to forced it to close the process. 

Second use the ROLE_ID in `npx directus users create --email admin@example.com --password password --role 9bc8c35d-6145-4595-8287-1895c747cb28`. 

After this I can login to my instance. I don't think this is a good solution, but a workaround to get started with directus 9.

I hope it will help you!
